commands/Fun/card.js

```diff
@@ -0,0 +1,37 @@
+const suits = ['♠️', '♦', '♥️', '♠️'];
+const ranks = ['A', '2', '3', '4', '5', '6', '7', '8', '9', '10', 'J', 'Q', 'K'];
+
+exports.run = (client, msg, [num]) => {
+  const numCards = num === undefined ? 1 : num;
+
+  if (numCards < 1 || numCards > 10) {
+    msg.channel.sendMessage('Number of cards should be between 1 and 10.');
+
+    return;
+  }
+
+  const lines = [];
+
+  for (let i = 0; i < numCards; ++i) {
```

Comment:
Should be i++ but following ESlint rules, Possibly changed to i =+ 1
Run it through the updated linter rules.